pkg/aspect/quotasManager.go

```diff
@@ -98,6 +100,24 @@ func (*quotasManager) ValidateConfig(c config.AspectParams, v expr.Validator, df
 		if _, err := quotaDefinitionFromProto(desc); err != nil {
 			ce = ce.Appendf(fmt.Sprintf("Descriptor[%s]", desc.Name), "failed to marshal descriptor into its adapter representation with err: %v", err)
 		}
+
+		if quota.MaxAmount < 0 {
+			ce = ce.Appendf("MaxAmount", "must be >= 0")
+		}
+
+		if quota.Expiration < 0 {
+			ce = ce.Appendf("Expiration", "cannot be less than 0")
+		}
+
+		if desc.RateLimit {
+			if quota.Expiration == 0 {
```

Comment:
I simplified it a bit. But I don't like your form, it requires a tiny bit too much thinking to make sure it's correct. I rather have a few more characters for extra ease of readability.